libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/impl/VideoCompressor.kt

```diff
@@ -29,7 +29,7 @@ class VideoCompressor @Inject constructor(
         val future = Transcoder.into(tmpFile.path)
             .setVideoTrackStrategy(
                 DefaultVideoStrategy.Builder()
-                    .addResizer(AtMostResizer(1920, 1080))
+                    .addResizer(AtMostResizer(720, 480))
```

Comment:
Note: the commit messages incorrectly says `720x48`, at least that's what I see on GH.

Reply:
Ah yes, that's a typo